dockertest/subtestbase_unittests.py

```diff
@@ -72,6 +74,36 @@ class DockerTestFail(Exception):
 ]
 
 
+class TestNotRedHat(TestCase):
+
+    def setUp(self):
+        import subtestbase
+        self.subtestbase = subtestbase
+        # Saves some typing
+        self.stbsb = self.subtestbase.SubBase
+        pfx = 'subtestbase_unitest'
+        (self.fd,
```

Comment:
Nope, then it would leak a file (at least until the unittest process exited.)  Docs state it's the caller's responsibility to close.